plugins/sdexFeed.go

```diff
@@ -0,0 +1,79 @@
+package plugins
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/interstellar/kelp/api"
+	"github.com/interstellar/kelp/support/utils"
+	"github.com/stellar/go/build"
+	"github.com/stellar/go/clients/horizon"
+)
+
+// sdexFeed represents a pricefeed from the SDEX
+type sdexFeed struct {
+	sdex       *SDEX
+	assetBase  *horizon.Asset
+	assetQuote *horizon.Asset
+}
+
+// ensure that it implements PriceFeed
+var _ api.PriceFeed = &sdexFeed{}
+
+// newSDEXFeed creates a price feed from buysell's url fields
+func newSDEXFeed(sdex *SDEX, url string) (*sdexFeed, error) {
+	s := new(sdexFeed)
+	s.sdex = sdex
+	urlParts := strings.Split(url, "/")
+
+	baseURL := strings.Split(urlParts[0], ":")
+	baseCode := baseURL[0]
+	baseIssuer := baseURL[1]
+	baseConvert, e := parseAsset(baseCode, baseIssuer)
+	if e != nil {
+		return nil, fmt.Errorf("unable to convert base asset url to sdex asset")
```

Comment:
we should never _hide_ errors, this should display the error: 

    return nil, fmt.Errorf("unable to convert base asset url to sdex asset: %s", e)

Reply:
Agreed, oversight on my part